### ISSUE TITLE
[UI/Feature Request] Add "Jump to Selected" button

### ISSUE THREAD

**Issue by idantene** [allegroai/clearml-server#141]
We usually end up with quite many experiments in a single project (easily over 100), and most of them are somehow related.
We also freely share the ClearML links (such a useful feature!), but then because the server shows the latest 15 experiments, we have to continuously press "Load More" just to get around to where the shared experiment is listed.

It would be amazing if a "Jump to Selected" button was available next to the "Load More" option 👍🏻

**Comment by ainoam** [comment]
Thanks for the feedback @idantene @juhoarjo!

| It would be even better if the selected experiment would be automatically loaded.

Isn't that already covered in the initial premise? i.e. the experiment that's **currently open** in the details panel is not on-screen in the experiment list

**Comment by idantene** [comment]
Exactly, it is currently not immediately available in the experiment list, and it would be useful if it were.
